pps_mw_training/utils/scaler.py

```diff
@@ -9,12 +9,13 @@
 
 
 @dataclass
-class Scaler:
-    """Scaler class."""
+class MinMaxScaler:
+    """Scaler class for Min Max Scaling"""
 
     xoffset: np.ndarray
     gain: np.ndarray
     ymin: np.ndarray
+    ymax: np.ndarray
```

Comment:
ymax is not used anywhere so I don't think we should have this attribute here.

Reply:
I removed the unncessary args and related functions